docs/getting_started.rst

```diff
@@ -33,8 +33,8 @@ Installation Quick Reference Guide
 
          python nengo_fpga/id_extractor.py <board>
 
-    where **<board>** is the board name as it appears in ``fpga_config``.
-    See the `Copy Protection`_ section for more information.
+   where **<board>** is the board name as it appears in ``fpga_config``.
```

Comment:
Should be tabbed over 1 more. It should be aligned to the i. and ii.

Reply:
I dont know why github is saying this comment is on an outdated file...

Reply:
so close haha